R/boundsPredprob.R

```diff
@@ -52,7 +52,7 @@ boundsPredprob <- function(looks, Nmax = max(looks), p0, tT, phiL, phiU, parE =
   if (missing(weights)) {
     weights <- rep(1, nrow(t(parE)))
   }
-  assert_numeric(weights, min.len = 0, len = nrow(par), finite = TRUE)
+  assert_numeric(weights, min.len = 0, len = nrow(parE), finite = TRUE)
```

Comment:
If `parE` is passed as a vector then this would fail, I guess we also need `t(parE)`, as in line 53?